doc/nrf/ug_nrf5340.rst

```diff
@@ -496,6 +496,49 @@ Follow these steps to build and program a multi-image build to the nRF5340 appli
 
 #. Program the application sample using :guilabel:`Target` > :guilabel:`Download zephyr/merged.hex`.
 
+Using |VSC| extension
+======================
+
+You can build and program separate images or combined images using |VSC|.
+
+Separate images
+---------------
+
+To build and program only the application core, follow the instructions in :`Building an application`_ and use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as build target.
+
+To build and program a dedicated network sample, follow the instructions in `Building an application`_ and use ``nrf5340dk_nrf5340_cpunet`` as the build target.
+
+.. _ug_nrf5340_VSC_multi_image:
+
+Multi-image build
+-----------------
+
+If you are working with Bluetooth LE, Thread, Zigbee, or Matter samples, the network core sample is built as child image when you build the application core image (see :ref:`ug_nrf5340_multi_image` above).
+
+Follow these steps to build and program a multi-image build to the nRF5340 application core and network core:
+
+1. Open Visual Studio code.
+#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar and then click :guilabel:`Open`.
+#. Click on :guilabel:`Add an existing application to workspace...`.
+#. In the prompt, navigate to the folder containing the sample you want to build, such as :file:`nrf/samples/bluetooth/peripheral_lbs`.
+   You should now see the selected application in the :guilabel:`Applications` window in the lower left corner.
```

Comment:
It seems to me that this window does not necessary have to be in the lower left year.
Meybe:
```suggestion
   You should now see the selected application in the :guilabel:`Applications` window of the NRF CONNECT panel.
```